blocks/hero-carousel/hero-carousel.css

```diff
@@ -0,0 +1,324 @@
+.section.hero-carousel-container {
+  padding: 0;
+}
+
+.section > .hero-carousel-wrapper {
+  max-width: 100%;
+}
+
+.hero-carousel .carousel-slides-container {
+  position: relative;
+  overflow: hidden;
+}
+
+.hero-carousel .carousel-slides,
+.hero-carousel .carousel-slide-indicators {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.hero-carousel .carousel-slides {
+  display: flex;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+  overflow: scroll clip;
+}
+
+.hero-carousel .carousel-slides::-webkit-scrollbar {
+  display: none;
+}
+
+.hero-carousel .carousel-slide.background-light-green {
+  background-color: #E3F6E9;
```

Comment:
use color variables , it looks like they are all defined in styles.css already :
  --swansdown: #e3f6e9;
  --skyblue: #def3fd;
etc.

Reply:
will take care of it in next iteration